docs/index.md

```diff
@@ -409,68 +431,37 @@ If you want to add another cluster (like asia-east1 in the above screenshots), r
     git push origin master
     ```
 
-1. Add the new cluster to the Shared CI/CD configuration repo:
-
-    ```shell
-    cd platform-admins/shared-ci-cd
-    export NEW_REGION="asia-east1"
-    cat >> cd/gke-deploy.yaml <<EOF
-    deploy-prod-$NEW_REGION:
-      # Only deploy when running on master in $APP-env
-      only:
-        refs:
-          - master
-      stage: deploy-prod-$NEW_REGION
-      image: gcr.io/cloud-builders/gke-deploy:stable
-      tags:
-      - cluster:prod-$NEW_REGION
-      script:
-        # Work around for https://github.com/kubernetes/kubernetes/issues/36072
-        # Stated solution is to use "--force" or "kubectl create"; gke-deploy does not have this option so the following works
-        # Get all services that have the "kubectl.kubernetes.io/last-applied-configuration" annotation and put into an environment variable
-        - export SERVICENAMES=$(kubectl get svc -o jsonpath="{range .items[?(@.metadata.annotations.kubectl\.kubernetes\.io/last-applied-configuration)]}{.metadata.name}{':'}{end}")
-        # for each service name, remove the annotation
-        - echo $SERVICENAMES | tr -d '\n' | tr ':' '\0' | xargs -0 -I {} kubectl annotate svc {} kubectl.kubernetes.io/last-applied-configuration-
-        - /gke-deploy run --filename prod.yaml
-    EOF
-    ```
-
-1. Now make similar changes in your `$APP_NAME-env` repo. This time we're going to make our changes in the `staging` branch, which already was created when we made our app above.
+1. Now add a new environment for the application using the cluster.
 
     ```shell
     export APP_NAME="go-app"
-    cd $APP_NAME-env
-    git checkout staging
-    git pull
-    cat >> .gitlab-ci.yml <<EOF
-    deploy-prod-$NEW_REGION:
-      tags:
-      - cluster:$SHORT_CLUSTERNAME, app:$APP_NAME
-    stages:
-    - deploy-$SHORT_CLUSTERNAME
-    EOF
+    export NEW_REGION="asia-east1"
+    export SHORT_CLUSTERNAME="prod-$NEW_REGION"
+    cd $APP_NAME
+    appctl env add $SHORT_CLUSTERNAME --cluster ${SHORT_CLUSTERNAME} --region ${NEW_REGSION} --namespace ${APP_NAME}
```

Comment:
We need to add instructions about how to install appctl.

Reply:
Before `appctl env add`, we should `appctl init` first.

Reply:
Shall we separate the change involving appctl into a different PR. I feel there are lots of places to change in this doc for appctl.

Reply:
reverted the change for this part. We can address it after the change for root repo.

Reply:
Agreed to move it to a separate PR.